spec/features/avo/search_with_policy_class_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.feature "Search with policy class", type: :system do
+  let(:url) { "/admin/resources/cities" }
+
+  subject do
+    visit url
+    page
+  end
+
+  describe "global search" do
+    let!(:city) { create :city, name: "São Paulo" }
+
+    context "when CustomCityPolicy allow searches" do
+      before do
+        ENV['ALLOW_CITY_INDEX'] = 'true'
+      end
+
+      it "can find the city" do
+        visit url
+        open_global_search_box
+        expect_search_panel_open
+
+        write_in_search "São Paulo"
+        expect(page).to have_content "São Paulo"
+        # expect(page).to have_link(href: "/any/custom/#{city.id}")
```

Comment:
I couldn't get that commented assert passing, I've seen this url in the search response but for somehow I can't see it in the page html during test

Reply:
You might need to regex it from the page.
```suggestion
        # expect(page).to have_link(href: /\/any/custom/#{city.id}/)
```

Reply:
The regex doesn't work also.

During the test, if I print `url` I can see `/any/custom/1`, but not in `page.body`, that's why the assert fail 🤔

Reply:
should I remove this commented line? Do you have any other suggestion on how to assert that?